src/sum_type.jl

```diff
@@ -35,7 +35,7 @@ function _sum_type(T, hidden, blk)
         i = findfirst(x -> x == p, T_params)
         T_params_constrained[i] isa Symbol && continue
         T_p_args = T_params_constrained[i].args
-        T_p_args[2] = :(Union{Uninit, $(T_p_args[2])})
+        T_p_args[2] = :(Union{$(SumTypes.Uninit), $(T_p_args[2])})
```

Comment:
```suggestion
        T_p_args[2] = :(Union{$Uninit, $(T_p_args[2])})
```
This is just stylistic, but the `SumTypes.` part isn't necessary here, since it's already interpolated.